packages/config-service/src/services/validationService.ts

```diff
@@ -51,8 +60,17 @@ export class ValidationService {
         case 'boolean':
           typeCastedEnvs[entryName] = envs[entryName] === 'true';
           break;
+        case 'array':
+          try {
+            const value = envs[entryName];
+            typeCastedEnvs[entryName] = value ? JSON.parse(value) : [];
+          } catch (e) {
+            // If parsing fails, fall back to the string value
+            typeCastedEnvs[entryName] = envs[entryName];
```

Comment:
this would defeat the purpose right? because we might have `string`s as well

Reply:
yes, it's fixed in my next commit